images/capi/Makefile

```diff
@@ -334,6 +334,8 @@ OCI_BUILD_NAMES			   ?= oci-ubuntu-1804 oci-ubuntu-2004 oci-ubuntu-2204 oci-orac
 
 DO_BUILD_NAMES 			?=	do-centos-7 do-ubuntu-1804 do-ubuntu-2004
 
+OPENSTACK_BUILD_NAMES	?=	openstack-ubuntu-2004	openstack-ubuntu-2204
```

Comment:
This isn't required due to the same entry 2 lines below this. By all means move them up so it's alphabetical, but it is repeating what is already there.

This has also been added to for the Flatcar additions provided in #6 and so can probably be removed.